src/xcode/ENA/ENA/Source/Extensions/__tests__/ENTemporaryExposureKey+ProcessingTests.swift

```diff
@@ -23,317 +23,422 @@ import ExposureNotification
 import XCTest
 
 final class ExposureKeysProcessingTests: XCTestCase {
-
-	func testSubmissionPreprocess_ApplyNoInformationVector() {
+	
+	func testSubmissionPreprocess_ApplyNoInformationVectors() {
 		let symptomsOnset: SymptomsOnset = .noInformation
 		let transmissionRiskVector = symptomsOnset.transmissionRiskVector
-
+		let daysSinceOnsetOfSymptomsVector = symptomsOnset.daysSinceOnsetOfSymptomsVector
+		
 		let days = 0...14
-		var keys = days.map { makeMockKey(daysUntilToday: $0) }
-		keys = keys.processedForSubmission(with: symptomsOnset, today: Date(timeIntervalSinceReferenceDate: 0))
-
-		keys.sort { $0.rollingStartNumber > $1.rollingStartNumber }
-
-		for (key, day) in zip(keys, days) {
+		let keys = days.map { makeMockKey(daysUntilToday: $0) }
+		var processedKeys = keys.processedForSubmission(with: symptomsOnset, today: Date(timeIntervalSinceReferenceDate: 0))
```

Comment:
Might be worth considering moving this section (until end of the for-loop) into a local helper-assertion to avoid code duplication in the code.

Reply:
I moved all the asserts to a helper function, makes the tests much more compact and readable, thanks for the suggestion!